content/rationale.md

```diff
@@ -0,0 +1,85 @@
+title: Why complate?
+
+Why complate?
+=============
+
+## Abstraction
+
+When we are developing our frontend components, we can use complate markup
+abstractions ("macros") as an abstraction layer for our component. Consider
+the following macro:
+
+```jsx
+function Navbar (_props, ...children) {
+  return <nav>
+    {children}
+  </nav>
+}
+```
+
+We can then use our macro in our application and it will be replaced with the
+markup we have defined.
+
+```jsx
+<Navbar />
+```
+
+This is _extremely powerful_ and solves a major pain point when writing
+maintainable frontend code. Requirements change constantly. When a new request
+comes, or the corporate design has changed yet again, I can change the markup
+of my component _without breaking any part of my application that uses the
+component_.
+
+Another major benefit is that it is possible to write complex frontend
+components and others can _use_ the components easily, without having to
+understand how the component works under the hood.
+
+## Composition
+
+complate allows us to write components and then explicitly compose them
+together. For instance, when I write a navbar component, I can reuse the
+component in multiple projects and change the content of the navbar without
+affecting it.
```

Comment:
I know what you mean, but "change the content without affecting it" sounds confusing - perhaps something along the lines of "without worrying about the surrounding boilerplate structures"?